pkg/ccl/backupccl/testdata/backup-restore/alter-schedule/missing-schedule

```diff
@@ -0,0 +1,74 @@
+new-server name=s1 allow-implicit-access
+----
+
+# Test that dropping the full backup causes a graceful failure on ALTER.
+
+exec-sql
+create schedule datatest for backup into 'nodelocal://1/example-schedule' recurring '@daily' full backup '@weekly';
+----
+
+let $fullID $incID
+with schedules as (show schedules) select id from schedules where label='datatest' order by command->>'backup_type' asc;
+----
+
+query-sql
+with schedules as (show schedules) select id, state, recurrence, owner, command from schedules where label='datatest' order by command->>'backup_type' asc;
+----
+$fullID <nil> @weekly root {"backup_statement": "BACKUP INTO 'nodelocal://1/example-schedule' WITH detached", "chain_protected_timestamp_records": true, "dependent_schedule_id": $incID, "unpause_on_success": $incID}
+$incID Waiting for initial backup to complete @daily root {"backup_statement": "BACKUP INTO LATEST IN 'nodelocal://1/example-schedule' WITH detached", "backup_type": 1, "chain_protected_timestamp_records": true, "dependent_schedule_id": $fullID}
+
+exec-sql
+drop schedule $fullID
+----
+
+# The schedule is now in an invalid state.
+
+query-sql
+with schedules as (show schedules) select id, state, recurrence, owner, command from schedules where label='datatest' order by command->>'backup_type' asc;
+----
+$incID Waiting for initial backup to complete @daily root {"backup_statement": "BACKUP INTO LATEST IN 'nodelocal://1/example-schedule' WITH detached", "backup_type": 1, "chain_protected_timestamp_records": true}
+
+exec-sql expect-error-regex=(pq: no full backup schedule found)
+alter backup schedule $incID set recurring '@hourly';
+----
+regex matches error
+
+exec-sql
+drop schedule $incID
+----
+
+# Test that dropping the incremental backup is not an issue.
+
+exec-sql
+create schedule datatest for backup into 'nodelocal://1/example-schedule' recurring '@daily' full backup '@weekly';
+----
+
+let $fullID $incID
+with schedules as (show schedules) select id from schedules where label='datatest' order by command->>'backup_type' asc;
+----
+
+query-sql
+with schedules as (show schedules) select id, state, recurrence, owner, command from schedules where label='datatest' order by command->>'backup_type' asc;
+----
+$fullID <nil> @weekly root {"backup_statement": "BACKUP INTO 'nodelocal://1/example-schedule' WITH detached", "chain_protected_timestamp_records": true, "dependent_schedule_id": $incID, "unpause_on_success": $incID}
+$incID Waiting for initial backup to complete @daily root {"backup_statement": "BACKUP INTO LATEST IN 'nodelocal://1/example-schedule' WITH detached", "backup_type": 1, "chain_protected_timestamp_records": true, "dependent_schedule_id": $fullID}
+
+exec-sql
+drop schedule $incID
+----
+
+# The schedule is now in an invalid state.
```

Comment:
nit: this is not an invalid state? inc being dropped is an acceptable state to be in.

Reply:
Copy-pasta, fixed. Thansk!